packages/suite-desktop/scripts/build.js

```diff
@@ -0,0 +1,22 @@
+/* eslint-disable */
```

Comment:
i would rater disable one rule (require) than the whole eslint

Reply:
That was just to get my IDE to stop complaining about these files (which shouldn't be linted). Added the `scripts` folder to the eslint ignore. 

3d8fd3b